### PR TITLE
[API-5536] - Update SpecialIssuesUpdater worker to use simple arguments

### PR DESCRIPTION
**Pull request by jw81**
## Description of change

Switches the ClaimsAPI SpecialIssuesUpdater worker to require simple arguments. This should prevent the problems that occur when marshalling/unmarshalling complex JSON objects into and out of REDIS.

## Original issue(s)
[API-5536](https://vajira.max.gov/browse/API-5536)

### CODE REVIEW

File: modules/claims_api/app/controllers/claims_api/v0/forms/disability_compensation_controller.rb [hunk 1]
```diff
@@ -35,7 +35,7 @@ def submit_form_526
             raise Common::Exceptions::UnprocessableEntity.new(detail: auto_claim.errors.messages.to_s)
           end
 
-          ClaimsApi::ClaimEstablisher.perform_async(auto_claim.id)
+          ClaimsApi::ClaimEstablisher.perform_async(auto_claim.id, target_veteran.participant_id)
```
Comment (SlakrHakr) [inline modules/claims_api/app/controllers/claims_api/v0/forms/disability_compensation_controller.rb:38]: you shouldnt need to pass this, i thought it was within the auto_claim headers

File: modules/claims_api/app/workers/claims_api/claim_establisher.rb [hunk 2]
```diff
@@ -68,18 +69,9 @@ def service(auth_headers)
...
-                            common_name: nil)
-      return user if auth_headers['va_bgs_authorization'].blank?
-
-      bgs_auth_headers = JSON.parse(auth_headers['va_bgs_authorization'])
-      user.uuid = bgs_auth_headers['external_uid']
-      user.email = bgs_auth_headers['external_key']
-
-      user
+      {
+        'ssn' => auth_headers['va_eauth_pnid']
```
Comment (SlakrHakr) [inline modules/claims_api/app/workers/claims_api/claim_establisher.rb:73]: pretty sure you just want to take the participant_id off the headers stored with the claim record and add it to this user object here, thats already getting passed into subsequent jobs

**Review by SlakrHakr — CHANGES_REQUESTED** [review]
everything looks good except id personally really like to see participant_id not being added as an argument to each of the jobs. shouldnt have to i dont think

**Comment by jw81** [comment]
@SlakrHakr I created a new branch off of Master and moved changes to over there. (rebasing this one off of Master was getting gross).  New PR can be found [here](https://github.com/department-of-veterans-affairs/vets-api/pull/6171).

**Pull request closed without merging**